### PR TITLE
Stack's cleaning #1

### PR DESCRIPTION
**Pull request by tintinmar1995**
* Delete everything related to Instance, Session, Subdomain, App, Job, Worker
* Clear some supposed-to-be useless packages 
* Delete the most part of routes and related packages
* Clear some tests

### CODE REVIEW

File: assets/templates/error.html [hunk 1]
```diff
@@ -4,12 +4,10 @@
...
-    {{.CozyUI}}
-    {{.ThemeCSS}}
-    <link rel="apple-touch-icon" sizes="180x180" href="{{asset .Domain "/apple-touch-icon.png" .ContextName}}">
-    <link rel="icon" type="image/png" href="{{asset .Domain "/favicon-32x32.png" .ContextName}}" sizes="32x32">
-    <link rel="icon" type="image/png" href="{{asset .Domain "/favicon-16x16.png" .ContextName}}" sizes="16x16">
+    <link rel="stylesheet">
+    <link rel="apple-touch-icon" sizes="180x180">
+    <link rel="icon" type="image/png" sizes="32x32">
+    <link rel="icon" type="image/png" sizes="16x16">
   </head>
```
Comment (paultranvan) [inline assets/templates/error.html:11]: I don't think this file is useful, as dispers's actors are not supposed to provide any UI

File: web/routing.go [hunk 2]
```diff
@@ -138,72 +82,28 @@ func SetupRoutes(router *echo.Echo) error {
...
+	/*
+		// non-authentified HTML routes
+		{
+			mws := []echo.MiddlewareFunc{
+				middlewares.Accept(middlewares.AcceptOptions{
+					DefaultContentTypeOffer: echo.MIMETextHTML,
+				}),
+				middlewares.CheckIE,
+			}
 		}
```
Comment (paultranvan) [inline web/routing.go:97]: Why the comments?

File: web/routing.go [hunk 3]
```diff
@@ -233,21 +134,19 @@ func SetupAdminRoutes(router *echo.Echo) error {
...
-	realtime.Routes(router.Group("/realtime", mws...))
 
 	setupRecover(router)
 
 	router.HTTPErrorHandler = errors.ErrorHandler
 	return nil
 }
 
-// CreateSubdomainProxy returns a new web server that will handle that apps
+// SetupMajorRoutes returns a new web server that will handle that apps
```
Comment (paultranvan) [inline web/routing.go:145]: There is no more apps

Comment (paultranvan) [reply web/routing.go:145]: Why major routes? There is no minor routes :)

File: assets/templates/error.html (hunk 1 above)
Comment (tintinmar1995) [reply assets/templates/error.html:11]: Understood. I wanted to keep the possibility to provide basic UI. I will delete everything about UI.

File: web/routing.go (hunk 2 above)
Comment (tintinmar1995) [reply web/routing.go:97]: It can be deleted. I kept that closely while fixing UI issues. It's useless now.

File: web/routing.go (hunk 3 above)
Comment (tintinmar1995) [reply web/routing.go:145]: Ok for the comment. 

It refers to "major" is server.go. I will refactor this function.

File: assets/templates/error.html (hunk 1 above)
Comment (paultranvan) [reply assets/templates/error.html:11]: Did you have a use-case in mind for the need of serving an UI?

Comment (tintinmar1995) [reply assets/templates/error.html:11]: Absolutely not

**Pull request closed without merging**